FactorioWebInterface/Program.cs

```diff
@@ -95,6 +96,8 @@ private static void SeedData(IHost host)
 
                 roleManager.CreateAsync(new IdentityRole(Constants.RootRole));
                 roleManager.CreateAsync(new IdentityRole(Constants.AdminRole));
```

Comment:
These need to be awaited to make sure the roles are present before calling SetupDefaultUserAsync. 

The SeedData function can be moved into the the WhenAll call on line 48.